packages/vue/rollup.config.js

```diff
@@ -4,13 +4,16 @@ export default {
   input: 'dist-transpiled/index.js',
   output: [
     {
-      file: 'dist/index.js',
-      format: 'esm',
+      dir: 'dist/',
+      entryFileNames: '[name].esm.js',
+      chunkFileNames: '[name]-[hash].esm.js',
+      format: 'es',
```

Comment:
My understanding is `format: 'es'` and `format: 'esm'` is going to do the same thing, so do we need to make any changes here?

Reply:
The "old" rollup.config.js outputs:
```
-rw-r--r--   1 zwacky  staff    55K Jan 14 17:05 index.js
-rw-r--r--   1 zwacky  staff   105K Jan 14 17:05 index.js.map
-rw-r--r--   1 zwacky  staff    23K Jan 14 17:05 index.min.js
-rw-r--r--   1 zwacky  staff    85K Jan 14 17:05 index.min.js.map
```

the "new" rollup.config.js in this PR outputs:
```
-rw-r--r--   1 zwacky  staff    55K Jan 14 17:09 index.esm.js
-rw-r--r--   1 zwacky  staff   105K Jan 14 17:09 index.esm.js.map
-rw-r--r--   1 zwacky  staff    25K Jan 14 17:09 index.js
-rw-r--r--   1 zwacky  staff    85K Jan 14 17:09 index.js.map
```

also in package.json it refers to:
```
...
"main": "./dist/index.js",
"module": "./dist/index.esm.js",
...
```

I hope I understood your question, this is not the area I daily work with.